source/transfer_list.rst

```diff
@@ -657,3 +658,59 @@ such that the last ACPI table in this entry ends at offset
 
 .. |hdr_size_desc| replace:: The size of this entry header in bytes must be set to **8**.
 .. |current_version| replace:: `0x1`
+
+
+.. _guid_blob_entry:
+
+GUID BLOB block entry layout (XFERLIST_GUID_BLOB)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This entry type holds an object identified by its GUID. The values and
+semantics should be consistent (but not limited to) with [UEFI]_ and [ACPI]_.
+
+.. _tab_guid_blob:
+.. list-table:: GUID BLOB block type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of `GuidBlob` (including BLOB content) in bytes.
+
+   * - guid_blob
+     - data_size
+     - hdr_size
+     - The guid_blob field contains a `GuidBlob`.
+
+.. code-block:: c
+
+        typedef struct {
```

Comment:
nit: C-style datastructure may lead to ambiguous datastrcuture alignment and size.
To clarify the alignment and field sizes, does it make sense to represent this as a list of fields with size and offset like we did on the other entries?

Reply:
I folded the content into the table.